apps/web/src/components/TweetInfo.tsx

```diff
@@ -194,7 +194,7 @@ const TweetInfo = ({ tweet }: TweetInfoProps) => {
       <Flex paddingY={3} borderY="1px solid" mt={4} borderColor="gray.300">
         <LinkBox>
           <LinkOverlay
-            href={`${twitterBaseUrl}${tweet.userInfo.username}/status/${tweet.id}${twitterEndpointSufix.retweet}`}
+            href={`${twitterBaseUrl}${tweet.userInfo.username}/status/${tweet.id}`}
```

Comment:
Why not here as well?

Reply:
The latter link leds to the actual post when clicked on replies number. Before this, it was leading to retweets page.